src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -198,7 +198,7 @@
 
     <ItemGroup>
       <_BuildSources Include="$(PrebuiltNuGetSourceName);$(PreviouslySourceBuiltNuGetSourceName);$(ReferencePackagesNuGetSourceName)"
-                     Condition="'$(DotNetBuildSourceOnly)' == 'true'"/>
+                     Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' != 'true'"/>
```

Comment:
Would this benefit from a boolean indirection property that determines whether prebuilt feeds should be added? i.e.

```
<IncludePrebuiltFeeds Condition="'$(DotNetSourceOnlyTestOnly)' == 'true'">false</IncludePrebuiltFeeds>
<IncludePrebuiltFeeds Condition="'$(IncludePrebuiltFeeds)' == '' and '$(DotNetBuildSourceOnly)' == 'true'">true</IncludePrebuiltFeeds>
```

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48327/commits/2eb97de97ca3407bf9e73841923d8ee8732d6a86